assets/app/scripts/controllers/overview.js

```diff
@@ -17,7 +17,8 @@ angular.module('openshiftConsole')
     $scope.displayRouteByService = {};
     $scope.unfilteredServices = {};
     $scope.deployments = {};
-    $scope.deploymentConfigs = {};
+    // Initialize to undefined so we know when deployment configs are actually loaded.
+    $scope.deploymentConfigs = undefined;
```

Comment:
do other parts of the template tolerate an undefined `deploymentConfigs`? I know we do things like `deploymentConfigs[deploymentConfigId]` without checking...